lib/encoder.py

```diff
@@ -731,10 +729,15 @@ def __init__(self, context):
   def WorkDir(self):
     return self.workdir
 
+  def SearchPath(self):
```

Comment:
Perhaps call this LocalAndImportedScorePaths or something similar? At the very least add a comment that  describes what this gives us.